standard_resources/revolve2/standard_resources/simulation.py

```diff
@@ -0,0 +1,64 @@
+"""Standard simulation functions and parameters."""
+
+from typing import List, Optional
+
+from revolve2.core.modular_robot import (
+    ModularRobot,
+    create_batch_multiple_isolated_robots,
+    create_batch_single_robot,
+)
+from revolve2.core.physics import Terrain
+from revolve2.core.physics.running import Batch
+
+STANDARD_SIMULATION_TIME = 30
+STANDARD_SAMPLING_FREQUENCY = 0.0001
+STANDARD_SIMULATION_TIMESTEP = 0.001
+STANDARD_CONTROL_FREQUENCY = 60
+
+
+def create_batch_single_robot_standard(
+    robot: ModularRobot,
+    terrain: Terrain,
+    simulation_time: Optional[int] = STANDARD_SIMULATION_TIME,
```

Comment:
is it intended not to expose `control_frequency` as in `create_batch_multiple_isolated_robots_standard`?

Reply:
I considered it not very important, but to be complete I will add it.